route.go

```diff
@@ -47,24 +48,24 @@ func (s *Server) shorten(w http.ResponseWriter, r *http.Request) {
 		fmt.Fprintf(w, "error: %v", e)
 		return
 	}
+	atomic.AddUint64(&s.Stats.TotalURL, 1)
 
 	fmt.Fprintf(w, "http://%v/%v", r.Host, key)
 }
 
 func (s *Server) statistics(w http.ResponseWriter, r *http.Request) {
-	s.updateStats(Statistics)
-
-	fmt.Fprintf(w, "URLs: %v\n", len(s.Kvs.storage))
-
-	fmt.Fprintf(w, "Redirect: %v\n", atomic.LoadUint64(&s.Stats.Redirects.Success))
-
-	fmt.Fprintf(w, "Handler:\n")
-	for _, v := range s.Stats.Handlers {
-		fmt.Fprintf(w, "  %v: %v\n", v.Name, atomic.LoadUint64(&v.Count))
+	atomic.AddUint64(&s.Stats.Handlers[Statistics].Count, 1)
```

Comment:
bonus point: can you think of a way to use an http middleware to handle in a single point the increment of the handler count metric?

You don't have to do in this PR, if you have some time between 2 exercices though you can have a look at https://about.sourcegraph.com/go/gophercon-2019-how-i-write-http-web-services-after-eight-years/ that explains clearly some interesting and idiomatic ways to create http servers in go, and middlewares.